### PR TITLE
Time to move surah info into database

### PR DESCRIPTION
**Pull request by naveed-ahmad**
## This PR will import static surah info into databse and provide api endpoint to access data for frontend app.

WIP for frontend. Going through huge learning curve :( 

### Steps to import data

* Merge this and deploy on staging app
* Ran the migration 
* `rake importer:import_surah_detail`

### CODE REVIEW

File: app/controllers/v2/surahs_controller.rb [hunk 1]
```diff
@@ -21,6 +21,16 @@ def show
     render json: surah
   end
 
+  api :GET, '/v2/surahs/:id/info', 'Get a details of specific Surah'
+  api_version '2.0'
+  param :id, :number
+  def info
+    #We have Surah info only in English for now. So no need to check locals from headers etc
+    surah_info = Quran::Surah.find(surah_params).get_surah_info_for_language('en')
```
Comment (sabeurthabti) [inline app/controllers/v2/surahs_controller.rb]: should the language be coming from a param?  or is that just a default? (my ruby is basic)

Comment (naveed-ahmad) [inline app/controllers/v2/surahs_controller.rb]: Sorry @sabeurthabti I did replied to your comment yesterday, but looks like it never goes through. 

We can use either params or headers to return language specific surah info. Unfortunately we don't have data for other language for now. So english is default until we deploy community app and have data for other languages. English is hard coded for now :) `get_surah_info_for_language('en')`

Comment (sabeurthabti) [inline app/controllers/v2/surahs_controller.rb]: can't we do something like `get_surah_info_for_language(param.lang||'en')` in ruby.

**Comment by mmahalwy** [comment]
Is `lib/tasks/importer.rake` needed anymore? Also, did you get a chance to push this to the db?

**Comment by naveed-ahmad** [comment]
@mmahalwy yes after build we need to run `rake importer:import_surah_detail` to import surah info into database. Or you want me to run this locally and push the database instead( weired and not recommended )

**Pull request merged**